maintainers/maintainer-list.nix

```diff
@@ -1641,6 +1641,12 @@
     githubId = 5561189;
     name = "Cody Opel";
   };
+  cohei = {
```

Comment:
Please make the maintainer list addition into a separate commit (`maintainers: add cohei`).

Reply:
👌